python/tank/authentication/user_impl.py

```diff
@@ -162,11 +171,13 @@ def __init__(self, host, login, session_token, http_proxy, password=None):
         :param session_token: Session token for the user. If session token is None
             the session token will be looked for in the users file.
         :param http_proxy: HTTP proxy to use with this host. Defaults to None.
+        :param password: Password for the user. Defaults to None.
```

Comment:
Thanks for picking up my slack. Seems like I forgot to document quite a few parameters back in the day.